packages/build/src/core/feature_flags.js

```diff
@@ -19,6 +19,8 @@ const getFeatureFlag = function(name) {
 }
 
 const DEFAULT_FEATURE_FLAGS = {
+  // When `true`, core plugins are run after user plugins instead of before
+  eventsOrder: false,
```

Comment:
[sand] Can we call this `runCorePluginsLast`?

Reply:
:+1: 

Fixed: 3bc8079a9de54265f4df650d7d589a6d16796750